geomstats/visualization.py

```diff
@@ -153,20 +180,34 @@ def plot(points, ax=None, space=None, **point_draw_kwargs):
                     'The plot function is not implemented'
                     ' for the space {}.'.format(space))
 
-        ax = plt.subplot(111, projection="3d", aspect="equal")
-        plt.setp(ax,
-                 xlim=(-ax_s, ax_s),
-                 ylim=(-ax_s, ax_s),
-                 zlim=(-ax_s, ax_s),
-                 xlabel="X", ylabel="Y", zlabel="Z")
+        if space is 'H2':
+            ax = plt.subplot(aspect='equal')
+            plt.setp(ax,
+                     xlim=(-ax_s, ax_s),
+                     ylim=(-ax_s, ax_s),
+                     xlabel='X', ylabel='Y')
+
+        else:
+            ax = plt.subplot(111, projection='3d', aspect='equal')
+            plt.setp(ax,
+                     xlim=(-ax_s, ax_s),
+                     ylim=(-ax_s, ax_s),
+                     zlim=(-ax_s, ax_s),
+                     xlabel='X', ylabel='Y', zlabel='Z')
 
     if space in ('SO3_GROUP', 'SE3_GROUP'):
         trihedrons = convert_to_trihedron(points, space=space)
         for t in trihedrons:
             t.draw(ax, **point_draw_kwargs)
+
     elif space is 'S2':
         sphere = Sphere()
         sphere.add_points(points)
         sphere.draw(ax)
 
+    elif space is 'H2':
```

Comment:
are you raising some ValueError if space is not in the 4 possibilities that you're passing?